src/build-scripts/build_materialX_osl.py

```diff
@@ -210,9 +210,13 @@ def main():
     parser.add_argument('-mx','--mx', default='../shaders/MaterialX', help='MaterialX source directory.  Default: ../shaders/MaterialX')
     parser.add_argument('-oslc_path', '--oslc_path', default='', help='Path to oslc executable.  Default: environment default')
     parser.add_argument('-compile', '--compile', default=0, help='Compile generated osl files 0|1.  Default: 0')
-    
+    parser.add_argument('-s', '--shader', default='', help='Specify a single mx shader to convert without the file extension, e.g. mx_add.  Default: none')
+    parser.add_argument('-t', '--types', default='', help='List of types to convert the mx file specified by the -s flag.  Default: all')
```

Comment:
This should be independent of -s 
This way someone can build all of the mx shaders but only for a specified subset of types.